lib/cmake/libcudacxx/libcudacxx-config-version.cmake

```
@@ -26,7 +26,7 @@ set(PACKAGE_VERSION_EXACT FALSE)
 set(PACKAGE_VERSION_UNSUITABLE FALSE)
 
 if(PACKAGE_VERSION VERSION_GREATER_EQUAL PACKAGE_FIND_VERSION)
-  if(PACKAGE_FIND_VERSION_MAJOR VERSION_EQUAL libcudacxx_VERSION_MAJOR)
+  if(PACKAGE_FIND_VERSION_MAJOR VERSION_LESS_EQUAL libcudacxx_VERSION_MAJOR)
```

```suggestion
  if(PACKAGE_FIND_VERSION_MAJOR VERSION_GREATER_EQUAL 1)
```

@robertmaynard I feel like this was a silly thing to write out, but if we ever decide `2.1.0` is our minimum between CUB and Thrust we can change this.